### ISSUE TITLE
Full Data Attributes Not in Response

### ISSUE THREAD

**Issue by netimpact1** [GSA/sam_api#135]
I am currently in the process of 'swapping out my soap requests to SAM' with the newly supported 'Rest API calls.'  As such, I am unable to get a response with a full data set per SAM record.  Previously all data attributes were returned when submitting up a SOAP request with multiple DUNS numbers. Is the ONLY workaround to submit a Rest API call one duns number at a time?

**Comment by aromero6** [comment]
Yes, the only way to return information on multiple DUNS is through multiple API calls.  As for the the data attributes, the SAM WebService or SOAP Request provides more data than the API.  The API is a public response system, where as the WebService is an authenticated service which will give more information and also has multiple role levels available, all which provide different sets of attributes.

Thank you and please let me know if you have any additional questions.

**Comment by netimpact1** [comment]
Thanks for validating my assumptions.  It appears as though the SOAP request is no longer an option?  Is WebService? If I could update multiple duns with a singular SOAP Request or SAM WS, I would be happy to do so but, could not find any documentation.
